### PR TITLE
Symlinks

### LINKED ISSUE #14: purpose of package?
I may be misunderstanding the purpose of this package. I had hoped that `conda install texlive-core` would install `latex` into my conda environment. In my situation, I need `latex` on TravisCI. So far, I've solved this by installing texlive through `apt-get`, but this is extremely slow. Therefore, I was hoping to install texlive through `conda` instead.

However, after installing `texlive-core`, there is no `latex` or `pdflatex` executable in the `bin` folder of my conda environment (there is a `tex` binary, however). Am I missing a step?

### PR DESCRIPTION
**Pull request by ocefpaf**
@bgruening can you take a quick look?

Closes #14

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,46 +1,49 @@
...
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: http://mirrors.concertpass.com/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
```
Comment (nsoranzo) [inline recipe/meta.yaml]: I'd revert this change, mirrors.ctan.org is automatically redirected to one of the many mirrors around the world according to the user position and is the link used at https://www.ctan.org/tex-archive/systems/texlive/Source

Comment (ocefpaf) [inline recipe/meta.yaml]: Thanks! I will modify that.

> I was working at the same PR today!!

Sorry about that. @bgruening made me feel bad that I did not act on #14 for so long :smile: 

PS: you should send a PR adding yourself as a maintainer though :wink:

Comment (nsoranzo) [inline recipe/meta.yaml]: Yeah, I noticed later that you got a ping from @bgruening because I need these symlinks for a Galaxy tool ;)

Comment (ocefpaf) [inline recipe/meta.yaml]: Travis-CI OS X machines seems to be down for today. If you need Linux we can merge this as is and, if things don't work on OS X, I can fix that later.

PS: should I add you as a maintainer in this PR?

Comment (nsoranzo) [inline recipe/meta.yaml]: No rush, but up to you.

That's a bit too far from my comfort zone, I'll have to decline the maintainer role, sorry.

BTW, is there a plan for additional TexLive packages? I'll need `xcolor` for example.

Comment (ocefpaf) [inline recipe/meta.yaml]: > That's a bit too far from my comfort zone

No problem. It is far from my comfort zone too :smile:

> BTW, is there a plan for how to contribute additional TexLive packages? I'll need xcolor for example.

We never settled on how to do that. A big bundle? Individual packages? Do you have any suggestion? I added this package mostly to help with `Jupyter` `nbconvert` stuff, so I never really though about the extras. (That is something I still rely on my systems package manager.)

Comment (nsoranzo) [inline recipe/meta.yaml]: Looking e.g. at https://src.fedoraproject.org/cgit/rpms/texlive.git/tree/texlive.spec gives me shivers. What could probably be done without too much work is create a `texlive` metapackage with 2 subpackages: `texlive-core` (containing more or less the same of this one) and `texlive-extras`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,46 +1,49 @@
...
+  number: 0
   skip: True  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
     - pkg-config
     - cairo 1.14.*
-    - gmp
+    - gmp >=5.0.1,<7
```
Comment (jakirkham) [inline recipe/meta.yaml]: This is now [`gmp 6.1.*`]( https://github.com/conda-forge/conda-forge.github.io/blob/7cecae63d1a62cedd9f4d1fce8b365d47efc6830/scripts/pin_the_slow_way.py#L50 ).

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,46 +1,49 @@
...
     - icu 58.*
-    - mpfr
+    - mpfr 3.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: This is now [`mpfr 3.1.*`]( https://github.com/conda-forge/conda-forge.github.io/blob/7cecae63d1a62cedd9f4d1fce8b365d47efc6830/scripts/pin_the_slow_way.py#L64 ).

**Comment by jakirkham** [comment]
The pinnings from the pinning PR were out-of-date. Have noted the updated pinnings in the review comments above. cc @gqmelo to double check in case I missed others.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,46 +1,49 @@
...
     - pixman 0.34.*
     - freetype 2.7|2.7.*
     - libpng >=1.6.28,<1.7
     - zlib 1.2.*
```
Comment (gqmelo) [inline recipe/meta.yaml]: zlib also tends to break compatibility between patch versions. So the only safe option is ```zlib 1.2.8```

**Comment by gqmelo** [comment]
@jakirkham it seems you only missed zlib ;). Thanks

**Comment by ocefpaf** [comment]
Thanks @gqmelo! BTW I am bringing one of these for you for all the contribution you gave to conda-forge so far.

conda-forge

**Comment by gqmelo** [comment]
@ocefpaf cool, thanks! But I'm afraid you won't be able to deliver it in time =/. I'm currently relocating to South Africa next week.

**Comment by ocefpaf** [comment]
Another one!? Joining Edison Gustavo?

**Comment by gqmelo** [comment]
yep!

**Comment by ocefpaf** [comment]
Good luck in this new endeavor!
